pkgs/development/compilers/rust/rustfmt.nix

```diff
@@ -0,0 +1,89 @@
+{ stdenv, rustPlatform, rustc, Security }:
+
+rustPlatform.buildRustPackage rec {
+  name = "rustfmt-${version}";
+  inherit (rustc) version src;
```

Comment:
Maybe use `rustPlatform.rust.rustc` instead of `rustc` ?

Reply:
Done. Thanks for the suggestion!